### ISSUE TITLE
Where is source code? Is project discontinued?

### ISSUE THREAD

**Issue by petriashev** [bherila/Swashbuckle.AspNetCore.AzureFunctions#8]
Source code was removed in last commit 960594325c5024c1f53a0245da3bc4978966b657.
Is project discontinued?

**Comment by IanKemp** [comment]
Yeah, I'm also confused about this one. Would appreciate feedback from @bherila as to what's going on.

**Comment by fgimian** [comment]
The README explanation is quite confusing as there's no link to the new repo and NuGet still points to this repository.  Based on whatever I can find, the code is now [here](https://github.com/bherila/Swashbuckle.AspNetCore).
